### ISSUE TITLE
lavPredict only works with > 2 rows of data

### ISSUE THREAD

**Issue by jackobailey** [yrosseel/lavaan#183]
I'm running lavaan 0.6-6 and trying to predict latent factor scores from a model I have fit conditional on setting certain parameters of interest (experimental treatments) to certain values. I'd like to pass a single row of data to `lavPredict`, but it doesn't work. Instead, it gives the following error: "Error in if (warn && max.var > 1e+06) { : missing value where TRUE/FALSE needed".

It seems that lavPredict will only work where you pass 3 or more rows of new data to the function.

Replication code below:

```
library(lavaan)

data("PoliticalDemocracy")

model <- '
  # measurement model
    ind60 =~ x1 + x2 + x3
    dem60 =~ y1 + y2 + y3 + y4
    dem65 =~ y5 + y6 + y7 + y8
  # regressions
    dem60 ~ ind60
    dem65 ~ ind60 + dem60
  # residual correlations
    y1 ~~ y5
    y2 ~~ y4 + y6
    y3 ~~ y7
    y4 ~~ y8
    y6 ~~ y8
'

fit <- sem(model, data=PoliticalDemocracy)

lavPredict(fit, type = "lv", newdata = PoliticalDemocracy) # Works
lavPredict(fit, type = "lv", newdata = PoliticalDemocracy[1, ]) # Doesn't work
lavPredict(fit, type = "lv", newdata = PoliticalDemocracy[1:2, ]) # Doesn't work
lavPredict(fit, type = "lv", newdata = PoliticalDemocracy[1:3, ]) # Works
```

**Comment by yrosseel** [comment]
Thanks for reporting this! This used to work, but additional checks (recently added) 'forgot' about this possibility. Fixed in lavaan dev 0.6-7.1575

**Comment by jackobailey** [comment]
Thanks, Yves!
